frontend/public/components/dashboard/dashboards-page/overview-dashboard/activity-card.tsx

```diff
@@ -115,7 +115,7 @@ const OngoingActivity = connect(mapStateToProps)(
               .filter((r) => (a.properties.isActivity ? a.properties.isActivity(r) : true))
```

Comment:
As commented above, `resource.data` is not always an array. In case when it is an object this throws runtime error:
```
TypeError: k8sResources.filter is not a function at activity-card.tsx:115
```

(same applies for the project-dashboard activity-card below)